modules/evm/rpc/runtime_api/src/lib.rs

```diff
@@ -49,7 +51,7 @@ sp_api::decl_runtime_apis! {
 			gas_limit: u64,
 			storage_limit: u32,
 			access_list: Option<Vec<AccessListItem>>,
-			estimate: bool,
+			rpc_mode: bool,
```

Comment:
the name is a bit confusing. what exactly does rpc_mode do? also it is false from estimate resources rpc so it is more confusing

Reply:
@xlc this needs to be true only when call coming from RPC `evm_call` so it will skip storage rent. For estimate it's ok to charge storage because execution of tx you're estimating will be the same when you submit it.

Reply:
@xlc I don't have a better name. Maybe `rpc_call` is better?

Reply:
`view_mode`?

Reply:
@xlc it's not just view. Maybe `rpc_call` is better. Indicating that the call is coming from rpc

Reply:
eth_call is view mode.
my point is, why disable storage charge when the call is coming from rpc? that's the part isn't explained just by reading the code.

Reply:
@xlc what about `ignore_storage: bool`?